libcob/common.c

```diff
@@ -3353,10 +3365,13 @@ cob_module_global_enter (cob_module **module, cob_global **mglobal,
 		cob_module_list = mod_ptr;
 	} else if (entry == 0
 		&& !cobglobptr->cob_call_from_c) {
```

Comment:
just stumbled over this, either the `&&` should have spaces before, not a tab, to indent "correct" or a line break be used after the `else` (we have this in several parts, not sure yet how I do like it [clangd doesn't understand that]

Reply:
Adjusted.